src/item_pocket.cpp

```diff
@@ -1299,6 +1299,7 @@ ret_val<item_pocket::contain_code> item_pocket::insert_item( const item &it )
     const ret_val<item_pocket::contain_code> ret = can_contain( it );
     if( contain_override || ret.success() ) {
         contents.push_back( it );
+        ret.make_success(); //may be bad for debugging but better than possible dublication
```

Comment:
Did you miss a `return` here? Currently this has no effect. `make_success` is a static function.